src/lib/homestore_backend/hs_pg_manager.cpp

```diff
@@ -174,14 +176,23 @@ HSHomeObject::HS_PG::HS_PG(PGInfo info, shared< homestore::ReplDev > rdev, share
         pg_sb_->members[i].priority = m.priority;
         ++i;
     }
-
     pg_sb_.write();
```

Comment:
since we use cp_flush to write pg_sb, do we need this write here?

Reply:
The idea is for PG creation (or via recovery), it will always be written in PG's constructor. After a particular PG is created, pg sb write always go through cp_flush, which is log protected.